test-runner/adapters/direct_python_sdk/__init__.py

```diff
@@ -4,12 +4,19 @@
 # Licensed under the MIT license. See LICENSE file in the project root for
 # full license information.
 
-from .direct_module_api import ModuleApi, object_list
+from .direct_module_api import ModuleApi, object_list_sync
+from .direct_module_api_aio import ModuleApiAsync, object_list_async
```

Comment:
>object_list_async [](start = 51, length = 17)

Something with the same name is imported twice (here and also on line 10). Like I said in a previous comment, these lists should be class properties instead to fully avoid this problem.